src/components/Button.js

```diff
@@ -2,18 +2,32 @@ import React from 'react';
 import PropTypes from 'prop-types';
 
 const Button = (props) => {
-  const { btnName } = props;
+  const { btnName, color, wide } = props;
+  let size = wide;
+  if (wide) {
+    size = '50%';
+  } else size = '25%';
+
   return (
-    <button type="button">{btnName}</button>
+    <button
+      type="button"
+      style={{ backgroundColor: color, width: size }}
+    >
+      {btnName}
+    </button>
   );
 };
```

Comment:
Few things you should consider doing:
- Destructure props directly in the parameter
``` js
const Button = ({ btnName, color, wide }) => {
```
- Use the `wide` prop directly by using the [tenary operator](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator).
```jsx
  return (
    <button
      type="button"
      style={{ backgroundColor: color, width: wide ? '50%' : '25%' }}
    >
      {btnName}
    </button>
  );
```